### PR TITLE
CPA-24 Get poems by author

### PR DESCRIPTION
**Pull request by yuriy1901**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ function App() {
 			<Modals />
 			<Routes>
 				<Route path="/" element={<HomePage />} />
-				<Route path="/author" element={<AuthorPage />} />
+				<Route path="/author/:authorName" element={<AuthorPage />} />
```
Comment (RudeAnna) [inline src/App.js:36]: could you add path to routes.js ROUTE_AUTHOR_PAGE

File: src/components/Poem/Poem.js [hunk 2]
```diff
@@ -12,18 +12,22 @@ import { Link } from "react-router-dom";
 const Poem = ({ poem }) => {
 	const navigate = useNavigate();
 
+	const linkToAuthorPage = () => {
```
Comment (MuscularSloth) [inline src/components/Poem/Poem.js:15]: Suggestion: Name of this function sounds like entity, but it should sounds like an action I think. For example `navigateToAuthorPage` or `goToAuthorPage` or smth else.

File: src/App.js (hunk 1 above)
Comment (RudeAnna) [inline src/App.js:36]: Could you add path to routes.js 
const ROUTE_AUTHOR_PAGE = "/author/:authorName"

Comment (RudeAnna) [reply src/App.js:36]: if I am not mistaken)

**Review by MuscularSloth — CHANGES_REQUESTED** [review]
Great job, but you are using an old version of the component Poem. We have several commits with new version for now. I suggest you fetch/merge main branch, solve all conflicts and then implement your changes.

File: src/App.js (hunk 1 above)
Comment (yuriy1901) [reply src/App.js:36]: Yes - will do it, thanks!

File: src/redux/poems/poemsActions.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import {
 	GET_POEMS_REQUEST,
 	GET_POEMS_SUCCESS,
 	GET_POEMS_SUCCESS_INFINITE_SCROLL,
+	GET_POEMS_BY_AUTHOR_SUCCESS
```
Comment (RudeAnna) [inline src/redux/poems/poemsActions.js:6]: Don't we need to have all three request, success, failure actions for each inquiry? like separate 
GET_POEMS_BY_AUTHOR_REQUEST
GET_POEMS_BY_AUTHOR_FAILURE
for tracking loading process and catching errors

Comment (yuriy1901) [reply src/redux/poems/poemsActions.js:6]: I am using existings methods for request and failure - becouse this is the same list. Only for success - i have other type

**Review by bejalane — APPROVED** [review]

**Review by MuscularSloth — APPROVED** [review]

**Pull request merged**